source/numerics.tex

```diff
@@ -7929,7 +7929,6 @@
 \begin{itemdecl}
 slice();
 slice(size_t start, size_t length, size_t stride);
-slice(const slice&);
```

Comment:
Weird that those were ever there! Why have no valarray users filed bug reports...

Reply:
> Why have no valarray users filed bug reports...

Maybe there are no valarray users?